packages/web/src/pages/upload-page/fields/SpecialAccessFields.module.css

```diff
@@ -0,0 +1,42 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: var(--unit-2);
+  padding: var(--unit-4);
+  background: var(--neutral-light-10);
+  border: 1px solid var(--neutral-light-7);
+  border-radius: var(--unit-2);
+  font-weight: var(--font-demi-bold);
+  font-size: var(--font-l);
+}
+
+.radio {
+  margin-right: var(--unit-1);
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  gap: var(--unit-2);
+}
+
+.row:not(.disabled) {
+  cursor: pointer;
+}
+
+.icon path {
+  fill: var(--neutral-light-4);
+}
+
+.tooltip :global(.ant-tooltip-inner) {
```

Comment:
it does feel sad having to hard-code all these random values into .ant tooltip :/